GitUI/Hotkey/HotkeySettingsManager.cs

```diff
@@ -322,6 +322,7 @@ public IReadOnlyList<HotkeySettings> CreateDefaultSettings()
                 new HotkeySettings(
                     FileViewer.HotkeySettingsName,
                     Hk(FileViewer.Command.Find, Keys.Control | Keys.F),
+                    Hk(FileViewer.Command.Replace, Keys.Control | Keys.H),
```

Comment:
Sort alphabetically, please.

Reply:
The existing commands are grouped by functionality.
Do you want to resort the existing commands too?
All in the file?

Reply:
OK, let's keep this single (reasonable) exception. (All other rubrics *are* in alphabetical order.)